### PR TITLE
deps: remove byte-size

### PR DESCRIPTION
**Pull request by wraithgar**
In its latest release, byte-size dropped support for node versions lower
than 14.  The cli currently supports node 10 and up.
    
The actual functionality we needed and was using was extremely limited
in scope, and didn't warrant an external module.  It's just pretty
printing a file size, and the files we are dealing with are limited in
size so we don't need to support so many suffixes.

### CODE REVIEW

File: lib/view.js [hunk 1]
```diff
@@ -356,7 +356,7 @@ class View extends BaseCommand {
         manifest.dist.integrity && color.yellow(manifest.dist.integrity),
       fileCount:
         manifest.dist.fileCount && color.yellow(manifest.dist.fileCount),
-      unpackedSize: unpackedSize && color.yellow(unpackedSize.value) + ' ' + unpackedSize.unit,
+      unpackedSize: unpackedSize && color.yellow(unpackedSize[0]) + ' ' + unpackedSize[1],
```
Comment (wraithgar) [inline lib/view.js]: Do we *really* really care about the unit not being colorized here?

Comment (isaacs) [reply lib/view.js]: That's a weird style choice, tbh.  I'd be fine with just making the whole thing yellow.

**Comment by wraithgar** [comment]
NOTE even pretty-bytes will soon not work in node10 so we may need to just implement this ourselves: https://github.com/sindresorhus/pretty-bytes/issues/68

**Comment by ljharb** [comment]
Seems like it might be better to just stay on v7 of byte-size in the meantime.

**Comment by wraithgar** [comment]
> Seems like it might be better to just stay on v7 of byte-size in the meantime.

The thing we're balancing this against is having outdated packages in the tree, we already have two that have pending reasons why they can't be updated at this time and every time the list is added to it becomes harder to manage.  We're trying to be pretty aggressive on this front.

**Comment by ljharb** [comment]
Understood, but it seems like adopting a package that is explicitly planning to be nonviable within the next 5 months is just hiding/punting the tech debt and difficulty, not reducing it.

**Comment by wraithgar** [comment]
Yes this is absolutely a calculated tech debt punting decision.

**Comment by wraithgar** [comment]
Rewrote this with @fritzy's help to just be the absolute minimum functionality we need, we're not using an entire formatting library here we're just translating a very small section of file size values.

File: lib/utils/tar.js [hunk 2]
```diff
@@ -28,8 +28,8 @@ const logTar = (tarball, opts = {}) => {
     { name: 'name:', value: tarball.name },
     { name: 'version:', value: tarball.version },
     tarball.filename && { name: 'filename:', value: tarball.filename },
-    { name: 'package size:', value: byteSize(tarball.size) },
-    { name: 'unpacked size:', value: byteSize(tarball.unpackedSize) },
+    { name: 'package size:', value: formatBytes(tarball.size) },
```
Comment (wraithgar) [inline lib/utils/tar.js:31]: I don't think these are actually used anywhere because byteSize would be returning an object here not a string.

**Review by isaacs — APPROVED** [review]
LGTM!

**Pull request merged**